CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2025.6.106 (18 July 2025) PreRelease
```

Comment:
Maybe we should delete the unrelease tags and releases for 104 and 105 and make the new release 2025.6.104?

Reply:
106 already out. just skipping the failed ones